homeassistant/components/mopar/lock.py

```diff
@@ -0,0 +1,74 @@
+"""Support for the Mopar vehicle lock."""
+import logging
+
+from homeassistant.const import STATE_LOCKED, STATE_UNLOCKED
+
+from homeassistant.components.lock import LockDevice
+
+from homeassistant.components.mopar import DOMAIN as MOPAR_DOMAIN, \
+    SUCCESS_RESPONSE
+
+REQUIREMENTS = ['motorparts==1.1.0']
```

Comment:
This is only needed in the component.

Reply:
We are importing from `motorparts` though. Is it assumed that because the component is loaded, it's requirements will be pulled in?

Reply:
Yes!